### PR TITLE
[Jetsnack] Improve RTL layouts

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt [hunk 1]
```diff
@@ -57,7 +57,7 @@ fun VerticalGrid(
             val columnY = Array(columns) { 0 }
             placeables.forEachIndexed { index, placeable ->
                 val column = index % columns
-                placeable.place(
+                placeable.placeRelative(
```
Comment (nickbutcher) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt:60]: We should probably remove this and use `LazyGrid` but not needed for this PR.

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt [hunk 2]
```diff
@@ -100,7 +102,10 @@ fun SnackCollection(
                 modifier = Modifier.align(Alignment.CenterVertically)
             ) {
                 Icon(
-                    imageVector = Icons.Outlined.ArrowForward,
+                    imageVector = mirroringIcon(
```
Comment (nickbutcher) [inline Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt:105]: Could this use `mirroringBackIcon`?

**Review by nickbutcher — APPROVED** [review]

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt (hunk 2 above)
Comment (JolandaVerhoef) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt:105]: Nope, in this layout the arrow points the other way.

See the forward pointing arrows in the screenshot:

File: Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt (hunk 1 above)
Comment (JolandaVerhoef) [reply Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt:60]: Good point!

**Pull request merged**